src/utils.cpp

```diff
@@ -77,6 +77,10 @@ static bool provider_cache_loaded = false;
 // Maps all providers of pid to vector of tracepoints on that provider
 static std::map<std::string, usdt_probe_list> usdt_provider_cache;
 
+static bool pid_in_different_mountns(int pid);
```

Comment:
I made this static for now because it hasn't been needed elsewhere, but this might be useful in other places in the code.